RefrigeratorExe/RefrigeratorExe/Item.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace RefrigeratorExe
+{
+    internal class Item : IComparable<Item>
+    {
+        private string _type;
+        private string _kosher;
+        private string _name;
+        private DateTime _date;
+        private int _space;
+        public Guid Id { get; }
+        public string Name
+        {
+            get
+            {
+                return _name;
+            }
+            private set
+            {
+                if (!value.Equals(""))
+                    _name = value;
+                else
+                    throw new ArithmeticException("Illegal name , name must contain a String ");
+            }
+        }
+
+        public Shelf ShelfItem { get; set; }
```

Comment:
Change name from ShelfItem to ShelfId. You dont need to save here the whole Shelf but only its Id or its floor number